core/remotes/docker/schema1/converter.go

```diff
@@ -69,7 +69,8 @@ type Converter struct {
 	layerBlobs map[digest.Digest]ocispec.Descriptor
 }
 
-var ErrDisabled = fmt.Errorf("pulling Schema 1 images have been deprecated and disabled by default since containerd v2.0. "+
+//nolint:staticcheck // ignore ST1005: error strings should not be capitalized
+var ErrDisabled = fmt.Errorf("Pulling Schema 1 images have been deprecated and disabled by default since containerd v2.0. "+
```

Comment:
This part of the change should also be squashed with the commit that changed it.